### PR TITLE
Bug 1875702: Add test to check network supported models

### PR DESCRIPTION
**Pull request by gouyang**
Add test to check network supported models are virtio and e1000e.
Add test to check model is disabled when sriov is selected.

Implementation for:
https://polarion.engineering.redhat.com/polarion/#/project/CNV/workitem?id=CNV-4780
https://polarion.engineering.redhat.com/polarion/#/project/CNV/workitem?id=CNV-4781

### CODE REVIEW

File: frontend/packages/kubevirt-plugin/integration-tests/tests/vm.resources.scenario.ts [hunk 1]
```diff
@@ -123,6 +124,16 @@ describe('Test network type presets and options', () => {
   const wizard = new Wizard();
   const leakedResources = new Set<string>();
 
+  const fillGeneralStep = async () => {
```
Comment (suomiy) [inline frontend/packages/kubevirt-plugin/integration-tests/tests/vm.resources.scenario.ts:127]: we could go a bit further. Can we split the processWizard logic into the steps and call processGeneralStep here instead?

we can also use getBasicVMBuilder() or one of the presets here

Comment (gouyang) [reply frontend/packages/kubevirt-plugin/integration-tests/tests/vm.resources.scenario.ts]: Updated.

File: frontend/packages/kubevirt-plugin/integration-tests/tests/vm.resources.scenario.ts [hunk 2]
```diff
@@ -123,6 +124,15 @@ describe('Test network type presets and options', () => {
   const wizard = new Wizard();
   const leakedResources = new Set<string>();
 
+  const vm = new VMBuilder()
```
Comment (suomiy) [inline frontend/packages/kubevirt-plugin/integration-tests/tests/vm.resources.scenario.ts:127]: it is easier to use getBasicVMBuilder here instead

File: frontend/packages/kubevirt-plugin/integration-tests/tests/vm.resources.scenario.ts (hunk 1 above)
Comment (suomiy) [reply frontend/packages/kubevirt-plugin/integration-tests/tests/vm.resources.scenario.ts]: Great, thanks!

File: frontend/packages/kubevirt-plugin/integration-tests/tests/vm.resources.scenario.ts (hunk 2 above)
Comment (gouyang) [reply frontend/packages/kubevirt-plugin/integration-tests/tests/vm.resources.scenario.ts]: use getBasicVMBuilder here now.

**Pull request merged**